storm-server/src/main/java/org/apache/storm/daemon/metrics/reporters/CsvPreparableReporter.java

```diff
@@ -18,14 +18,18 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
+
+import org.apache.storm.DaemonConfig;
 import org.apache.storm.daemon.metrics.ClientMetricsUtils;
 import org.apache.storm.daemon.metrics.MetricsUtils;
+import org.apache.storm.utils.ObjectReader;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class CsvPreparableReporter implements PreparableReporter {
     private static final Logger LOG = LoggerFactory.getLogger(CsvPreparableReporter.class);
     CsvReporter reporter = null;
+    Integer reportingIntervalPeriod = null;
```

Comment:
here too: It would be better to change reportingIntervalPeriod to reportingIntervalSecs